src/components/index.tsx

```diff
@@ -1,7 +1,10 @@
 import { Box, Container } from "@mui/material";
 import { OTPInput } from "../molecules/OTPInput";
-
+import VoiceRecorder from '../molecules/VoiceRecorder'
```

Comment:
hey move the VoiceRecorder to component section.

Reply:
@geeky-abhishek isnt voiceRecorder a molecule ? Should I move molecules form molecule section to component section